### PR TITLE
Finish static pages

### PR DESCRIPTION
**Pull request by nguyenthetoan**

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (phannh58) [inline app/assets/javascripts/static_pages.coffee]: remove file

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (phannh58) [inline app/assets/stylesheets/static_pages.scss]: remove file

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (phannh58) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
+                                              'data-turbolinks-track' => true %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (phannh58) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
```
Comment (phannh58) [inline app/views/static_pages/about.html.erb]: i18n

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %>
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
```
Comment (phannh58) [inline app/views/static_pages/home.html.erb]: i18n

File: config/routes.rb [hunk 7]
```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
```
Comment (phannh58) [inline config/routes.rb]: double quote

File: config/routes.rb [hunk 8]
```diff
@@ -1,3 +1,9 @@
...
+
+  get 'static_pages/home'
```
Comment (phannh58) [inline config/routes.rb]: double quote

File: config/routes.rb [hunk 9]
```diff
@@ -1,3 +1,9 @@
...
+
+  get 'static_pages/help'
```
Comment (phannh58) [inline config/routes.rb]: double quote

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (phanbt58) [inline app/views/layouts/application.html.erb]: convention, 'data-turbolinks-track' :  true

Comment (phanbt58) [inline app/views/layouts/application.html.erb]: convention:  'data-turbolinks-track' :  true

File: app/views/static_pages/about.html.erb [hunk 10]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```
Comment (phanbt58) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/static_pages/about.html.erb [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
+</p>
```
Comment (phanbt58) [inline app/views/static_pages/about.html.erb]: I18n, chuyển hết các text vào file en.yml

File: app/views/layouts/application.html.erb [hunk 12]
```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```
Comment (nhunght) [inline app/views/layouts/application.html.erb]: remove ()
i18n

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (nhunght) [inline app/views/layouts/application.html.erb]: media: :all

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (nhunght) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track": true

File: app/views/static_pages/about.html.erb [hunk 13]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, t(".title") ) %>
+<h1>About</h1>
+<p>
+ <%= t (".the") %> <a href="http://www.railstutorial.org/"><em><%= t (".rort") %></em></a> <%= t (".is") %>
+ <a href="http://www.railstutorial.org/book"><%= t (".book") %></a> <%= t (".and") %>
+ <a href="http://screencasts.railstutorial.org/"><%= t (".screencasts") %></a>
+ <%= t (".teach") %>
+ <a href="http://rubyonrails.org/"><%= t (".ror") %></a>.
+ <%= t (".p") %>
+</p>
```
Comment (phannh58) [inline app/views/static_pages/about.html.erb]: convention

File: app/views/static_pages/help.html.erb [hunk 14]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "Help") %>
+<h1>Help</h1>
+<p>
+  <%=t (".p1")%>
+  <a href="http://www.railstutorial.org/#help"><%=t (".p2") %></a>.
+  <%=t (".p3") %>
+  <a href="http://www.railstutorial.org/book"><em><%=t (".rort")%></em>
+  <%=t (".book")%></a>.
+</p>
```
Comment (phannh58) [inline app/views/static_pages/help.html.erb]: convention

File: app/views/static_pages/home.html.erb [hunk 15]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, t(".title")) %>
+<h1>Sample App</h1>
+<p>
+ <%=t (".phome") %>
+ <a href="http://www.railstutorial.org/"><%=t (".rort")%></a>
+ sample application.
+</p>
```
Comment (phannh58) [inline app/views/static_pages/home.html.erb]: convention

**Pull request closed without merging**